### PR TITLE
add task solution

### PR DESCRIPTION
**Pull request by SerjDSA**
[DEMO LINK](https://serjdsa.github.io/js_2048_game/)

### CODE REVIEW

**Review by etojeDenys — CHANGES_REQUESTED** [review]
1. why do you show zero on empty cells, let's just make the empty cell like in reference
image


2. when we don't have any cells to merge then do not add another one

https://github.com/mate-academy/js_2048_game/assets/70220667/5fe924b8-9abe-45e6-9033-f2e4ccd1a944

**Review by lerastarynets — CHANGES_REQUESTED** [review]
You still have zeros in cells and new number appear on every arrow button click, even if cells can't be moved/merged

**Review by SerjDSA — COMMENTED** [review]
[DEMO LINK](https://serjdsa.github.io/js_2048_game/)

File: src/index.html [hunk 1]
```diff
@@ -14,40 +21,57 @@ <h1>2048</h1>
...
-            <td class="field-cell"></td>
-            <td class="field-cell"></td>
+            <td class="field-cell"
+            id="cell-1-0"></td>
+            <td class="field-cell"
+            id="cell-1-1"></td>
+            <td class="field-cell"
+            id="cell-1-2"></td>
+            <td class="field-cell"
+            id="cell-1-3"></td>
```
Comment (DarkMistyRoom) [inline src/index.html:50]: fix the code style

**Review by DarkMistyRoom — CHANGES_REQUESTED** [review]
- fix please these issues [video1](https://www.loom.com/share/cec3857c087547e4951d1aac89fb01df?sid=df1f92e0-e2fd-4193-9c87-33e532d3785f)

[video2](https://www.loom.com/share/9b11906869154effb0266c7a9d8f47f7?sid=dd50e8b1-06f1-4fcc-ba8c-65bc0a99dfd1)

- why do I lose? There are tiles that could be merged
image

**Review by lerastarynets — CHANGES_REQUESTED** [review]
I keep loosing, while there are still cells that can be merged
Screenshot at Jan 15 14-45-23

**Review by SerjDSA — COMMENTED** [review]
[DEMO LINK](https://serjdsa.github.io/js_2048_game/)

**Review by etojeDenys — CHANGES_REQUESTED** [review]
still can reproduce it
https://github.com/mate-academy/js_2048_game/assets/70220667/5f8c57da-1515-4b68-acae-312350b3b767

**Review by etojeDenys — CHANGES_REQUESTED** [review]
https://github.com/mate-academy/js_2048_game/assets/70220667/24db0dae-f78e-4570-b088-54f9707fc9c5

feel free to ask for some help in the chat, if you don't know how to fix that

**Review by SerjDSA — COMMENTED** [review]
[DEMO LINK](https://serjdsa.github.io/js_2048_game/)

**Review by lerastarynets — CHANGES_REQUESTED** [review]
Pls watch the video review [via the link](https://www.awesomescreenshot.com/video/24074540?key=7dc0afa129d4923bf2535c9846f9e755) :)

**Review by SerjDSA — COMMENTED** [review]
[DEMO LINK](https://serjdsa.github.io/js_2048_game/)

**Pull request closed without merging**